CONTRIBUTING.adoc

```diff
@@ -269,181 +225,122 @@ This is an *AsciiDoc* formatted bio, but it is completely optional!
-  <<jenkins-user-documentation,Jenkins User Documentation>> section below for
-  details on how this content is structured.
-. *Extend Jenkins Documentation* - this documentation is for people who want to
-  _extend_ the functionality of Jenkins by developing their own Jenkins plugins.
-  Like the Jenkins User Documentation (above), the content is based on the same
-  link:https://www.divio.com/blog/beginners_guide_to_documentation/[documentation model]. The
-  content for this set of documentation is written up as a combination of
-  `.haml` and `.adoc` files located in the
-  link:content/doc/developer[`content/doc/developer/`]
-  directory. Read more about adding pages to this documentation in
-  <<adding-a-stand-alone-page,Adding a stand-alone-page>>.
-. *Solution pages* - topic-specific destination pages providing a high-level
-  overview of a topic with links into getting started guides, handbook chapters,
-  relevant plugins and multimedia related to the topic. Be aware that some of
-  this content might already be present in the Jenkins User / Extend Jenkins
-  Documentation.
-
-The documentation pages can use the same metadata (`title`, `description`, `opengraph:image`)
-as blog posts.
+. *Jenkins User Documentation:* For people who want to _use_ Jenkins's existing functionality and plugin features.
+The documentation model that the content is based on is described in Michael Nicholson's blog post "link:https://www.divio.com/blog/beginners_guide_to_documentation/[Beginner's Guide to Documentation: Here's What You Need to Know]".
+Refer to the <<jenkins-user-documentation,Jenkins User Documentation>> section below for details on how this content is structured.
+. *Extend Jenkins Documentation:* This documentation is for people who want to _extend_ the functionality of Jenkins by developing their own Jenkins plugins.
+Like the Jenkins User Documentation (above), the content is based on the same link:https://www.divio.com/blog/beginners_guide_to_documentation/[documentation model].
+The content for this set of documentation is written up as a combination of `.haml` and `.adoc` files located in the link:content/doc/developer[`content/doc/developer/`] directory.
+Read more about adding pages to this documentation in <<adding-a-stand-alone-page,Adding a stand-alone-page>>.
+. *Solution pages:* Topic-specific destination pages providing a high-level overview of a topic with links into getting started guides, handbook chapters, relevant plugins, and multimedia related to the topic.
+Be aware that some of this content might already be present in the Jenkins User / Extend Jenkins Documentation.
+
+The documentation pages can use the same metadata (`title`, `description`, `opengraph:image`) as blog posts.
+
+==== Adding images
+
+When you add screenshots or images to documentation, there are methods to ensure that the images are focused, clear, and useful to the reader:
```

Comment:
Docs Office Hours (Asia) noted that it would be good to insert an example image markup into this text.  Alternately, link to the asciidoc documentation so that the writer can see multiple examples of image placement on the page, text wrapping, alt text, and how to implement the practices described in this guide.